03/rhys/tags.py

```diff
@@ -0,0 +1,61 @@
+from collections import Counter
+from difflib import SequenceMatcher
+from itertools import product
+import re
+
+IDENTICAL = 1.0
+TOP_NUMBER = 10
+RSS_FEED = "rss.xml"
+SIMILAR = 0.87
+TAG_HTML = re.compile(r"<category>([^<]+)</category>")
+
+
+def get_tags():
+    """Find all tags (TAG_HTML) in RSS_FEED.
+    Replace dash with whitespace.
+    Hint: use TAG_HTML.findall"""
+    with open("rss.xml", "r") as f:
+        for line in f:
+            line = line.replace("-", " ").lower()
+            tags = TAG_HTML.findall(line)
```

Comment:
will this override `tags` in each loop?

you can do it once, no loop: `TAG_HTML.findall(f.read().lower())`